### PR TITLE
Added docs for drone-rancher-stack-v1 plugin

### PR DESCRIPTION
**Pull request by wayneconnolly**
Please merge this plugin contribution that enables Rancher v1 (1.6) stack deployments/upgrades.

### CODE REVIEW

**Comment by tboerger** [comment]
This documentation doesn't match the standard plugin index Format at all.

**Comment by wayneconnolly** [comment]
Is there a url to show how to match that format?

**Comment by appleboy** [comment]
@wayneconnolly You can refer another PR.

**Comment by wayneconnolly** [comment]
Thanks for the tips. I have updated the index.md accordingly https://github.com/drone/drone-plugin-index/pull/136/commits/e2de0ad5f9f5f3d38ada1fbebd3fbb13b6bbf78e

**Comment by wayneconnolly** [comment]
When will this be merged ?

File: content/wayneconnolly/drone-rancher-stack-v1/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+date: 2018-07-02T00:00:00+00:00
+title: rancher-stack-v1
```
Comment (tboerger) [inline content/wayneconnolly/drone-rancher-stack-v1/index.md]: The title should be something human readable like Rancher Stack v1.

File: content/wayneconnolly/drone-rancher-stack-v1/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    url: rancher.domain.com
+    stack: app-staging        
+    accesskey: RANCHER_ACCESS_KEY
+    secretkey: RANCHER_SECRET_KEY
+    pull: true
+```
+
+# Parameter Reference
+
+image
```
Comment (tboerger) [inline content/wayneconnolly/drone-rancher-stack-v1/index.md]: Image is not really a parameter for the plugin, please drop it.

File: content/wayneconnolly/drone-rancher-stack-v1/index.md (hunk 1 above)
Comment (wayneconnolly) [reply content/wayneconnolly/drone-rancher-stack-v1/index.md]: I have updated as per your suggestion. Re: secrets: I don't provide secrets. If you want to do that then Ill have to update the image. I'm happy to as secrets should be used. Actually - gimme a day to do that. I'm at work now so can't test.

File: content/wayneconnolly/drone-rancher-stack-v1/index.md (hunk 2 above)
Comment (wayneconnolly) [reply content/wayneconnolly/drone-rancher-stack-v1/index.md]: Dropped

**Comment by tboerger** [comment]
Thanks for the changes, an example for the secrets you can see at https://github.com/drone/drone-plugin-index/blob/master/content/drone-plugins/drone-download/index.md#secret-reference

File: content/wayneconnolly/drone-rancher-stack-v1/index.md (hunk 1 above)
Comment (tboerger) [reply content/wayneconnolly/drone-rancher-stack-v1/index.md]: No problem, but you should always allow secrets for credentials ;)

**Comment by tonglil** [comment]
Merging without the secret reference changes.

**Pull request merged**